### ISSUE TITLE
iOS: Dialogs become invisible after opening and closing a QLPreview (only iOS 9) [duplicate of #84]

### ISSUE THREAD

**Issue by BeatKe** [aritchie/userdialogs#93]
I had the exactly same problem as mikey-mike in the issue #84.
But reinstalling all packages and cleaning up my project didn't solve the problem for me.

About the problem: All dialoges works fine, util I open a file in a QLPreview. After closing this preview all dialogs become invisible. But they still exist and worked.
To get the dialogs back I had to open the keyboard. For example by focusing to a textbox.

I resolved this problem by overriding your "UserDialogsImpl.cs" and use is as a custom instance in iOS.

```C#
protected override void Present(UIAlertController alert) {
            UIApplication.SharedApplication.InvokeOnMainThread(() => {
                UIWindow window = UIApplication.SharedApplication.KeyWindow;
                UIViewController viewController = window.RootViewController;
                if (viewController == null) {
                    while (viewController.PresentedViewController != null) {
                        viewController = viewController.PresentedViewController;
                    }
                }
                if (alert.PopoverPresentationController != null) {
                    var x = viewController.View.Bounds.Width / 2;
                    var y = viewController.View.Bounds.Bottom;
                    var rect = new CGRect(x, y, 0, 0);

                    alert.PopoverPresentationController.SourceView = viewController.View;
                    alert.PopoverPresentationController.SourceRect = rect;
                    alert.PopoverPresentationController.PermittedArrowDirections = UIPopoverArrowDirection.Unknown;
                }
                viewController.PresentViewController(alert, true, null);
            });
        }
```

**Comment by aritchie** [comment]
Try nuget release 4.1.6

**Comment by aritchie** [comment]
A fix was applied in 4.1.6.  Haven't heard back on the result so I am closing this defect
